dlt/dlt.py

```diff
@@ -1039,6 +1050,33 @@ def client_loop(self):
         dltlib.dlt_client_main_loop(ctypes.byref(self), None, self.verbose)
 
 
+def py_dlt_file_main_loop(dlt_reader, limit=None, callback=None):
```

Comment:
Please use the same function signature with `py_dlt_client_main_loop`

Reply:
just added docstring